apps/shared/avifutil.c

```diff
@@ -220,6 +224,37 @@ avifAppFileFormat avifGuessFileFormat(const char * filename)
     return AVIF_APP_FILE_FORMAT_UNKNOWN;
 }
 
+avifAppFileFormat avifReadImage(const char * filename,
+                                avifPixelFormat requestedFormat, int requestedDepth,
+                                avifImage *image, uint32_t *outDepth) {
```

Comment:
Please run clang-format. Use the clang-format command line in https://github.com/AOMediaCodec/libavif/blob/main/README.md.

Reply:
[The clang-format-check GitHub action](https://github.com/vrabaud/libavif/runs/7217915080?check_suite_focus=true#step:4:10) did not catch the issue. There might be some configuration error, maybe because of the `**` patterns?

`/entrypoint.sh: line 44: local: 'apps/shared/iccjpeg.h': invalid variable name for name reference` also puzzles me.

Reply:
dir/**.h does not take subdirectories.

Reply:
Done